runtime/lua/vim/lsp/util.lua

```diff
@@ -1481,6 +1481,25 @@ function M.open_floating_preview(contents, syntax, opts)
   return floating_bufnr, floating_winnr
 end
 
+local function convert_utf_to_byte(line, idx, offset_encoding)
+  -- convert to 0 based indexing
+  idx = idx - 1
+  P(offset_encoding)
```

Comment:
?

Reply:
oh haha sorry, forgot to remove this leftover